synthetics-stats/src/entities/fees.ts

```diff
@@ -114,6 +114,7 @@ export function savePositionFeesInfo(
   feesInfo.positionFeeAmount = eventData.getUintItem("positionFeeAmount")!;
   feesInfo.borrowingFeeAmount = eventData.getUintItem("borrowingFeeAmount")!;
   feesInfo.fundingFeeAmount = eventData.getUintItem("fundingFeeAmount")!;
+  feesInfo.liquidationFeeAmount = eventData.getUintItem("liquidationFeeAmount")!;
```

Comment:
liquidationFeeAmount is not available in events prior to newly created ones,
thus exclamation mark is redundant here

Reply:
I haven't check UI but fyi, we can't guruantee that liquidationFeeAmount would be available in tradeAction for the same mentioned reason

Reply:
make sense, made optional